app/achievements/models.py

```diff
@@ -12,19 +13,21 @@ def create_user(self, code: str):
             auth = create_auth_handler()
             auth.get_auth_token(code)
             client = Client(auth)
+            if settings.OSU_DEV_SERVER:
+                client.set_domain("dev.ppy.sh")
```

Comment:
can just remove this since it'll set domain based on the auth object

Reply:
ok